server/storage/wal/metrics.go

```diff
@@ -28,6 +28,15 @@ var (
 		Buckets: prometheus.ExponentialBuckets(0.001, 2, 14),
 	})
 
+	walWriteSec = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "etcd",
+		Subsystem: "disk",
```

Comment:
why disk subsystem?

Reply:
not so sure, do you have any suggestion, since I saw `backend_commit_write_duration_seconds` in _server\storage\backend\metrics.go_ is also labeled as disk subsystem, which contains both write and fdatasync process

Reply:
to be precise, maybe we should use _file_system_ subsystem?

Reply:
"disk" should be OK. It's consistent with the existing metrics `walWriteBytes`,

https://github.com/etcd-io/etcd/blob/1348062c23440a5f08b9834e427855a14af371e0/server/storage/wal/metrics.go#L31-L33

https://github.com/etcd-io/etcd/blob/1348062c23440a5f08b9834e427855a14af371e0/server/storage/wal/encoder.go#L95-L96